oar/cli/cmd_change_advisory_status.py

```diff
@@ -0,0 +1,39 @@
+import click
+import logging
+from oar.core.worksheet_mgr import WorksheetManager
+from oar.core.advisory_mgr import AdvisoryManager
+from oar.core.config_store import ConfigStore
+from oar.core.const import *
+
+logger = logging.getLogger(__name__)
+
+
+@click.command()
+@click.pass_context
+@click.option(
+    "-s",
+    "--status",
+    default=AD_STATUS_REL_PREP,
+    help="Valid advisory status, default is REL_PREP",
+)
+def change_advisory_status(ctx, status):
+    """
+    Change advisory status e.g. QE, REL_PREP
```

Comment:
seems qe cannot change advisory to QE,  Just move to REL_PREP

Reply:
it does not matter. the func is just used to change the status. the cmd can be used to change advisory status from `NEW_FILES` to `QE` as well.